### DISCUSSION TITLE
Proposal: Darwinia Bridge Enhancement with xToken BAAS (Bridge as a Service) by HelixDAO

### DISCUSSION THREAD

**Discussion by 1022ranji** [darwinia-network/darwinia#1479] [General]
<p>Dear Darwinia Community,</p>
<p>We hereby submit our proposal for funding support to cover the integration, setup, and ongoing maintenance costs for xToken BAAS (Bridge as a Service) in Darwinia. This service is designed to streamline interoperability and enhance the overall efficiency and utility of the Darwinia Network.</p>
<h2>Service Details</h2>
<p><a href="https://bridge.darwinia.network/">xToken BAAS by HelixDAO has been integrated with Darwinia</a>:</p>
<ul>
<li><strong>Integration and Setup of xToken BAAS:</strong> To integrate xToken BAAS with Darwinia, providing a robust and secure bridge service that enhances user experience and network capability, including darwinia&lt;&gt;ethereum, darwinia&lt;&gt;crab xtoken bridge service.</li>
<li><strong>Operational Support:</strong> Ensuring continuous and reliable service, providing indexing and front-end services, including regular updates and maintenance of the bridge infrastructure.</li>
<li><strong>Technical Support:</strong> Offering comprehensive technical assistance to ensure the network's uninterrupted functioning.</li>
</ul>
<h2><strong>Summary of Current Charges</strong></h2>

Description | Billing Period | Subtotal(USD)
-- | -- | --
xToken BAAS: Integration and set up fee (40% discount during the promotional period) | - | 20,000
xToken BAAS: Operation and Maintenance | 2024-04-01 to 2025-03-31 | 2,000

<h2>Funding Request</h2>
<p>We are seeking funding from the Darwinia Treasury to cover the initial integration and setup fee, as well as the first year’s operational costs as outlined above. The total funding request for this proposal is $22,000 USD, which will ensure that xToken BAAS can be successfully implemented and maintained, thereby benefiting the Darwinia community and ecosystem.</p>
<p>Total: $22,000 =  4,943,820 RING</p>
<p>RING Token Price (7 Day Average) = $0.00445</p>
<p>HelixDAO Account (Darwinia Chain): 0x000000000f681D85374225EdEeADC25560C1fb3F</p>
<p><a href="https://prod-files-secure.s3.us-west-2.amazonaws.com/19f4d35d-1c9c-4a17-ada7-7061951b94b0/b292a21c-23ef-4c19-9215-96e6d119895d/RING_7D_graph_coinmarketcap.csv">RING_7D_graph_coinmarketcap.csv</a></p>
<p>For any further details or clarifications, please do not hesitate to contact us. Thank you for your attention and support!</p>
<p>Sincerely,</p>
HelixDAO

**Comment by 2xJDubs** [comment]
Absolutely support this proposal.  With Helix moving to a standalone project like Subscan did a few years ago, it needs to be funded by projects using it services.  Good to see this progression in Helix's development.

**Comment by SasoLithops** [comment]
Absolutely, with no doubt on this proposal from Helix DAO. 

xToken BASS is amazing product from what I heard from Aki's presentation.
